src/components/shared.module.css

```
@@ -186,3 +187,19 @@
   border-radius: 2px;
   box-shadow: 0 0 0 3px color-mix(in srgb, var(--accent-primary) 24%, transparent);
 }
+
+@media (max-width: 767px) {
+  :global(.mobile-tab-panel) .toggle-btn,
```

Stylelint (17.7.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)